source/_docs/mqtt/broker.markdown

```diff
@@ -92,6 +92,30 @@ mqtt:
   certificate: /home/paulus/downloads/mosquitto.org.crt
 ```
 
+### HiveMQ Cloud
+
+HiveMQ Cloud is a fully managed MQTT broker that provides you a private broker. A free plan for up to 100 devices is available. You can see our different plan options here: https://www.hivemq.com/mqtt-cloud-broker/
```

Comment:
Don't use bare URLs.

Additionally: "You can see our different plan options here"

"our"? Are you affiliated with HiveMQ?